### PR TITLE
lmctl compatibility change corresponding to restconf driver

### LINKED ISSUE #132: RestConf compatibility with lmctl
Currently, lmctl supports ansible, kubernetes, sol003 & sol005 drivers to create the resource package with the help of lmctl command.
Now, we have applied the consolidated changes which will allow to create a resource package for restconf driver as well.
PR is already raised for the changes.
#133

### PR DESCRIPTION
**Pull request by rkumar8j**
Fixes #132

### CODE REVIEW

File: lmctl/project/handlers/brent/brent_src.py [hunk 1]
```diff
@@ -144,6 +144,7 @@ class AnsibleLifecycleTree(files.Tree):
 
     CONFIG_DIR_NAME = 'config'
     SCRIPTS_DIR_NAME = 'scripts'
+    TEMPLATE_DIR_NAME = 'template'
```
Comment (dvaccarosenna) [inline lmctl/project/handlers/brent/brent_src.py:147]: Added to the `AnsibleLifecycleTree`? Is this needed?

File: lmctl/project/handlers/brent/brent_src.py [hunk 2]
```diff
@@ -505,8 +527,36 @@ def __create_lifecycle(self, journal, source_request, file_ops, source_tree, lif
                 ptype='string')
             descriptor.add_property('localizationLanguage', description='Localization language of the NS to be instantiated', ptype='string')
             descriptor.insert_lifecycle('Create')
-            descriptor.insert_lifecycle('Install')
-            descriptor.insert_lifecycle('Uninstall')
+            descriptor.insert_lifecycle('Update')
```
Comment (dvaccarosenna) [inline lmctl/project/handlers/brent/brent_src.py:530]: These update the lifecycles included when generating a sol005 resource - was it intentional to change these?

**Review by dvaccarosenna — CHANGES_REQUESTED** [review]

File: lmctl/project/handlers/brent/brent_src.py (hunk 1 above)
Comment (rkumar8j) [reply lmctl/project/handlers/brent/brent_src.py]: This is typo mistake when I was merging the code line by line. Just committed the fixed code.

File: lmctl/project/handlers/brent/brent_src.py (hunk 2 above)
Comment (rkumar8j) [reply lmctl/project/handlers/brent/brent_src.py]: It happened when I was merging the code line by line. Just committed the fixed code.

**Review by dvaccarosenna — APPROVED** [review]

**Pull request merged**